pkg/apic/provisioning/schemabuilder.go

```diff
@@ -1,11 +1,19 @@
 package provisioning
 
-import "encoding/json"
+import (
+	"encoding/json"
+
+	"github.com/Axway/agent-sdk/pkg/util/log"
+)
+
+// propertyOrderSet - global flag to check if property order was set
+var propertyOrderSet = false
```

Comment:
This should not be global, it should be part of the schemabuilder struct

Reply:
done...